CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## master / unreleased
+* [ENHANCEMENT] Upgraded Docker base images to `alpine:3.17`. #5132
```

Comment:
nit: The order is CHANGE, FEATURE, ENHANCEMENT, BUGFIX. 
We have not been paying attention.... 😄 

can you at least put it next to the other ENHANCEMENTS?

Reply:
Got you :)

I moved it to the right place. Thanks